src/states/game.ts

```diff
@@ -3,7 +3,7 @@ import Player from '../components/Player';
 import Enemy from '../components/Enemy';
 import Powerup from '../components/Powerup';
 import TankrApp from '../app';
-import {Images} from '../assets';
+import {Audio, Images} from '../assets';
```

Comment:
it would be nice to have audio for barrel destruction too, since it makes a big kaboom

Reply:
Done